internal/cli/internal/command/volumes/list.go

```diff
@@ -0,0 +1,84 @@
+package volumes
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/dustin/go-humanize"
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/pkg/iostreams"
+
+	"github.com/superfly/flyctl/internal/cli/internal/app"
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/config"
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/cli/internal/render"
+	"github.com/superfly/flyctl/internal/client"
+)
+
+func newList() *cobra.Command {
+	const (
+		long = "List all the volumes associated with this application."
+
+		short = "List the volumes for app"
+	)
+
+	cmd := command.New("list", short, long, runList,
+		command.RequireSession,
+		command.RequireAppName,
+	)
+
+	flag.Add(
+		cmd,
+		flag.App(),
+		flag.AppConfig(),
+	)
+
+	return cmd
+}
+
+func runList(ctx context.Context) (err error) {
```

Comment:
same comment as with other function. decide whether you're going named return parameters or not and format the rest of the function accordingly, please.